src/utils/boardIdUnserializer.ts

```diff
@@ -0,0 +1,6 @@
+import { makeMarshal } from '@endo/marshal';
+
+/**
+ *  Unserializes values from the board as just their ID strings.
+ */
+export const boardIdUnserializer = makeMarshal(undefined, slot => slot);
```

Comment:
this is really a whole `Marshaller`. It has `serialize` and `unserialize` methods.

Also this utility module should be more general so it can export more than one thing. I'd suggest `src/utils/updates.js` to put all the non-UI parts of `App.tsx`.

Reply:
Just renamed to identyMarshal and refactored non-ui code as suggested,